.pre-commit-config.yaml

```diff
@@ -5,6 +5,12 @@ repos:
       - id: pyupgrade
         args: ["--py37-plus"]
 
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v2.0.0
+    hooks:
+      - id: autoflake
+        args: ["--in-place", "--imports"]
```

Comment:
> -i, --in-place        make changes to files instead of printing diffs

Cool

> --imports IMPORTS     by default, only unused standard library imports are removed; specify a comma-separated list of additional modules/packages

What does the `--imports` here is supposed to affect, based on the above doc?
The default should work without en empty `--imports`, no?

>  --remove-unused-variables
                        remove unused variables

This might also be interesting, but I'm not 100% sure.